doc/langref.html.in

```diff
@@ -11267,7 +11261,7 @@ all your base are belong to us{#end_shell_samp#}
       {#header_close#}
       {#header_close#}
       {#header_open|WebAssembly#}
-      <p>Zig supports building for WebAssembly out of the box.</p>
+      <p>Zig natively supports building for WebAssembly.</p>
```

Comment:
I think the intention for "out of the box" is to mean "without any external dependencies" (besides the unpacked Zig distribution).
I'm not sure on the precise definition of "native", but I end up thinking about the user's operating and CPU - which, if Zig is a native program, it operates natively, but that would still be achieved if using external programs.
Therefore I don't see this being an improvement, personally.

Reply:
"out of the box" is a bit of indirect slang , so I thought it would be clearer to remove it.  But you're right about "natively" being overloaded in this context, so I'll remove this change.